nvflare/private/fed/client/client_executor.py

```diff
@@ -393,6 +395,31 @@ def _wait_child_process_finish(self, client, job_id, allocated_resource, token,
             return_code = get_return_code(job_handle, job_id, workspace, self.logger)
 
             self.logger.info(f"run ({job_id}): child worker process finished with RC {return_code}")
+
+            should_report_error_log = (
+                False  # set this to True to report error log to server, todo: get value from client config
```

Comment:
Do we plan on implement this in this PR?

Reply:
Not this one, this will be implemented in a future PR